apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -231,37 +230,83 @@ private void Write(IEnumerable<LogEntry> logEntries)
             DateTimeRange logEntriesLost;
             lock (_enqueueLock)
             {
-                if (_initNextIdTask != null)
+                if (_initIdTask != null)
                 {
-                    _initNextIdTask.Wait();
-                    _initNextIdTask.Dispose();
-                    _initNextIdTask = null;
+                    _currentId = _initIdTask.Result ?? -1;
+                    _initIdTask.Dispose();
```

Comment:
You don't really need to dispose of tasks..
https://blogs.msdn.microsoft.com/pfxteam/2012/03/25/do-i-need-to-dispose-of-tasks/